backend/app/api/v1/github.py

```
@@ -0,0 +1,34 @@
+        issues = await issue_service.fetch_beginner_issues(
+            owner=GITHUB_ORG,
+            repo=repo
+        )
+
+        return {
+            "repo": repo,
+            "count": len(issues),
+            "issues": issues
+        }
+
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)